F17_lab03.md

```diff
@@ -0,0 +1,77 @@
+#### Wei Tung Chen github: lleeoochen
+#### Nicholas Duncan github: nduncan1014
+#### Mentor: natashalee@umail.ucsb.edu
+
+#### (a) (20 pts) A brief description of the project. Here, I’m looking for a short description: probably 1 sentence, 2-3 at most. 
+
+* This is a remake of the original Pac-Man arcade game which involves moving a character through a maze of pellets whilst avoiding the enemy ghosts. The goal is to collect all of the pellets without losing all lives.
+
+#### (b) (20 pts) a set of user stories (as a X I can Y so that Z) that describe what the current software in its current state can do.
+
+* As a player, I can press arrow keys so that pacman moves around the maze, and eat pellets, powerups and avoid ghosts. As a player, I can choose co-op mode to have two pacmans on screen.
+* As a player, I can choose versus mode, which allows users to control both pacman and ghosts. As a player I can enter in my score after a game finishes so that I can keep track of my high scores.
```

Comment:
Did you guys mean "as a player I can enter in my _name_ so I can keep track of my high scores?"